app/helpers/validation_helper.php

```diff
@@ -27,8 +27,36 @@ function validate_existence($check, $type)
     return empty($check) ? true : User::isValidUsernameEmail($check, $type);
 }
 
+function validate_content($check)
+{
+    return empty($check) ? false : true;
```

Comment:
Change po to ``` return trim($check); ``` https://github.com/renzosunico/MyClassroom/blob/20631e2d7cc07d79d1a4e38fe8c1a89b8f6d0d27/app/helpers/validation_helper.php Because the result is inverted po. :bow: